### DISCUSSION TITLE
Feature Suggestion: Image Carousel (for Vue3)!

### DISCUSSION THREAD

**Discussion by mgav** [tailwindlabs/headlessui#1434] [Ideas]
Thanks for all of your terrific work!

**Comment by seanonthenet** [comment]
+1 Carousel for React 🙏

**Comment by ghost** [comment]
There is already one general requested, let's push it forward: #1011 Vue and React together let's hope we get stuff out :)

**Comment by jiblett1000** [comment]
Any reason why the tabs component couldn't be used for this? I'm not seeing much of a difference in terms of functionality...
